openapi-maven-plugin/src/main/java/io/github/kbuntrock/model/OperationType.java

```diff
@@ -30,7 +30,14 @@ public enum OperationType {
 	private final Class javaxRsVerbAnnotation;
 	private final String jakartaRsVerbAnnotation;
 
-	OperationType(final RequestMethod springMvcRequestMethod, final Class javaxRsVerbAnnotation, final String jakartaRsVerbAnnotation) {
+	/**
+	 * Restitution order in the generated documentation for same urls
+	 */
+	private final int schemaOrder;
```

Comment:
You don't need `schemaOrder`, all java enums are comparable by default: https://docs.oracle.com/javase/6/docs/api/java/lang/Enum.html